### DISCUSSION TITLE
Is there a way to do exclude on specific image repo ?

### DISCUSSION THREAD

**Discussion by dolevhadad** [sigstore/cosign#2687] [Q&A]
Hi everyone 
I traying to deploy Cosing by helm in my cluster and enable it on all of my namespaces by label.
I need to exclude one of the images in my pod (istio) that was set up by TAG and not by SHA.
Currently i can't convert the tag to SHA in the pods istio containers.

_The error that i get is:_
**Error creating: admission webhook "cosigned.sigstore.dev" denied the request: validation failed: invalid value: *************.dkr.ecr.eu-west-1.amazonaws.com/containers.istio.tetratelabs.com/proxyv2:1.13.5-tetratefips-v0 must be an image digest: spec.containers[1].image, spec.initContainers[0].image**

Is there a way that can I exclude the image repo and tag?

**Comment by Hayden-IO** [comment]
Closing since this has been open for awhile. If you have questions still, please ask on the Sigstore cosign slack channel.
